openedx/core/djangoapps/credentials/tests/factories.py

```diff
@@ -0,0 +1,53 @@
+"""Factories for generating fake credentials-related data."""
+import factory
+from factory.fuzzy import FuzzyText
+
+
+class UserCredentials(factory.Factory):
+    """Factory for stubbing user credentials resources from the User Credentials
+    API (v1).
+    """
+    class Meta(object):
+        model = dict
+
+    id = factory.Sequence(lambda n: n)  # pylint: disable=invalid-name
+    username = FuzzyText(prefix='user_')
+    status = 'awarded'
+    uuid = FuzzyText(prefix='uuid_')
+    certificate_url = 'http=//credentials.edx.org/credentials/dummy-uuid'
+    credential = {}
+
+
+class ProgramCredentials(factory.Factory):
+    """Factory for stubbing program credentials resources from the Program
+    Credentials API (v1).
+    """
+    class Meta(object):
+        model = dict
+
+    credential_id = factory.Sequence(lambda n: n)
+    program_id = factory.Sequence(lambda n: n)
+
+
+class CourseCredentials(factory.Factory):
+    """Factory for stubbing course credentials resources from the Course
+    Credentials API (v1).
+    """
+    class Meta(object):
+        model = dict
+
+    course_id = 'edx/test01/2015'
+    credential_id = factory.Sequence(lambda n: n)
+    certificate_type = 'verified'
+
+
+class ProgramCredential(factory.Factory):
```

Comment:
Is this factory necessary? It seems strange to use one for this. The structure of this data is an artifact of a utility function.